sklearn/ensemble/tests/test_forest.py

```diff
@@ -1359,3 +1361,19 @@ def test_multi_target(name, oob_score):
     # Try to fit and predict.
     clf.fit(X, y)
     clf.predict(X)
+
+
+def test_forest_feature_importances_sum():
+    X, y = make_classification(n_samples=15, n_informative=3, random_state=1,
+                               n_classes=3)
+    clf = RandomForestClassifier(min_samples_leaf=5, random_state=42,
+                                 n_estimators=200).fit(X, y)
+    assert math.isclose(1, clf.feature_importances_.sum(), abs_tol=1e-7)
```

Comment:
Just curious, why not `np.is_close()`?

Reply:
numpy's `isclose` is _older_ than python's `math.isclose` and has some issues compared to the python's. Here's a nice conversation on the topic: https://github.com/numpy/numpy/issues/10161. Since now we support python>=3.5, seems like a good choice to me to use `math.isclose` if comparing two floats is what we want to do.